ambuda/checks.py

```diff
@@ -107,7 +106,7 @@ def _check_app_schema_matches_db_schema(database_uri: str) -> list[str]:
 
 
 def _check_lookup_tables(database_uri: str) -> list[str]:
-    engine = create_engine(database_uri)
+    _ = create_engine(database_uri)
```

Comment:
I think we can delete this line entirely -- it's just a copy-paste from the function above, and it's basically a no-op here.